src/VisualStudio/CSharp/Impl/CSharpVSResources.resx

```diff
@@ -516,4 +516,7 @@
   <data name="When_possible" xml:space="preserve">
     <value>When possible</value>
   </data>
+  <data name="Warn_on_invalid_string_dot_format_calls" xml:space="preserve">
+    <value>Warn on invalid string.Format method calls</value>
```

Comment:
When they open their 15 year old project that they are doing maintenance on, and the error list is full of reports of things that they don't care about right now.